bootstrap-joining-demo/aws-api-ecs/src/main/scala/akka/discovery/awsapi/ecs/AsyncEcsSimpleServiceDiscovery.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.discovery.awsapi.ecs
+
+import java.net.{InetAddress, NetworkInterface}
+import java.util.concurrent.TimeoutException
+
+import akka.actor.ActorSystem
+import akka.discovery.SimpleServiceDiscovery
+import akka.discovery.SimpleServiceDiscovery.{Resolved, ResolvedTarget}
+import akka.discovery.awsapi.ecs.AsyncEcsSimpleServiceDiscovery._
+import akka.pattern.after
+import software.amazon.awssdk.services.ecs._
+import software.amazon.awssdk.services.ecs.model._
+
+import scala.collection.JavaConverters._
+import scala.collection.immutable.Seq
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.duration._
+import scala.concurrent.{ExecutionContext, Future}
+
+// This is duplicated from the discovery-aws-api-async (for now...).
+// TODO: Remove when 0.11.0 is released.
+class AsyncEcsSimpleServiceDiscovery(system: ActorSystem)
```

Comment:
It would be nice to avoid doing this (even though it is temporary). I'm open to suggestions for alternative approaches!

Reply:
I was thinking the same, but then I thought there must be a reason the EC2 bootstrap demo did it this way. Without knowing if that was the case and what the reason was, I thought it better to stick with consistency for this PR.

I'd be happy to unify it all into one build. Most of my opportunity for working on this has been at weekends though and I'm away all of the next weekend, so I'm not certain I'd do it promptly.

Reply:
@ktoso @dhpiggott you are right - the PR that I have open (that adds integration testing EC2 tag based) makes the EC2 bootstrap demo a subproject of the main one :-)